packages/office-ui-fabric-react/src/components/List/List.tsx

```diff
@@ -11,6 +12,7 @@ import {
   IRenderFunction
 } from '../../Utilities';
 import { IList, IListProps, IPage, IPageProps, ScrollToMode } from './List.types';
+import { EventGroup } from '@uifabric/utilities';
```

Comment:
Shouldn’t this be imported from ../../Utilities by consolidating it with line 2 import?

Reply:
https://github.com/OfficeDev/office-ui-fabric-react/pull/9810/commits/33959e998710239251a98e706e7f35c1040f3b46